extensions/installed-extensions/src/index.tsx

```diff
@@ -52,13 +52,14 @@ export default function IndexCommand() {
     let result = await Promise.all(
       files.map(async (file) => {
         const content = await fs.readFile(file, "utf-8");
+        const stats = await fs.stat(file);
         const json = await JSON.parse(content);
```

Comment:
This `await` for the `JSON.parse` is extraneous

Reply:
Thanks for noticing 👏